app/models/api_docs/backend_api.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+class ApiDocs::BackendApi < ApplicationRecord
```

Comment:
Why not using the same model by adding a polymorphic owner, like we did for `Metric` and `ProxyRule`?